contracts/SortitionPool.sol

```diff
@@ -73,29 +108,24 @@ contract SortitionPool is SortitionTree, Ownable {
   {
     uint256 weight = getWeight(authorizedStake);
 
+    Rewards.updateOperatorRewards(operator, uint32(weight));
+
     if (weight == 0) {
       _removeOperator(operator);
     } else {
       updateOperator(operator, weight);
     }
   }
 
-  /// @notice Removes an operator from the pool.
-  /// @dev Can be called only by the contract owner.
-  /// @param operator Address of the operator to be removed.
-  function removeOperator(address operator) public onlyOwner onlyUnlocked {
-    _removeOperator(operator);
-  }
-
-  /// @notice Ban rewards for given operators for given period of time.
-  /// @dev Can be called only by the contract owner.
-  /// @param operators IDs of banned operators.
-  /// @param duration Duration of the ban in seconds.
-  function banRewards(uint32[] calldata operators, uint256 duration)
-    external
+  function setRewardIneligibility(address[] calldata operators, uint256 until)
```

Comment:
We need this function to accept `uint32[]` identifiers to integrate it with the rest of the code.

Reply:
Addressed 👍